src/edu/csus/ecs/pc2/core/execute/SandboxUtilities.java

```diff
@@ -23,28 +23,35 @@
      * 
```

Comment:
The first line of JavaDoc comments should end in a period.  More importantly, it's not at all clear what the comment means -- what does it mean to "write input"?  It would be quite helpful if there was some kind of explanation here...

Reply:
no change, will do doc improvements later.   This is prototype code not final code.